dynamiqs/integrators/sepropagator/diffrax_integrator.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import diffrax as dx
+
+from ..core.abstract_integrator import SEPropagatorIntegrator
+from ..core.diffrax_integrator import (
+    DiffraxIntegrator,
+    Dopri5Integrator,
+    Dopri8Integrator,
+    EulerIntegrator,
+    Kvaerno3Integrator,
+    Kvaerno5Integrator,
+    Tsit5Integrator,
+)
+
+
+class SEPropagatorDiffraxIntegrator(DiffraxIntegrator, SEPropagatorIntegrator):
+    @property
+    def terms(self) -> dx.AbstractTerm:
+        # define Schrödinger term d|psi>/dt = - i H |psi>
```

Comment:
I would change to 
```suggestion
        # define Schrödinger term dU/dt = - i H U
```
in this context